team-props/git-hooks.gradle

```
@@ -35,12 +34,21 @@ tasks.register('installGitHooks', Exec) {
     description = "Installs the pre-commit git hooks from team-props/git-hooks."
     group = "git hooks"
     workingDir = rootDir
-    commandLine "chmod", "-R", "+x", ".git/hooks/"
+
+    // Ensure commandLine is always set
+    if (isWindows()) {
+        // Log or handle the Windows case if needed
+        logger.lifecycle("Skipping chmod on Windows")
+        commandLine 'cmd', '/c', 'echo', "Git hooks installed (chmod skipped on Windows)" // Placeholder command
+    } else {
+        commandLine "chmod", "-R", "+x", ".git/hooks/"
+    }
+
     doLast {
         logger.info("Git hook installed successfully!")
     }
 }
 
 afterEvaluate {
     clean.dependsOn installGitHooks
-}
+}
```

File content must end with a blank line 
```suggestion
}

```